src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -106,41 +115,49 @@ class Hovercard extends React.Component {
       },
     };
 
+    const visible = show !== undefined ? show : this.state.visible;
+    const hoverProps =
+      show !== undefined
+        ? {}
+        : {onMouseEnter: this.handleToggleOn, onMouseLeave: this.handleToggleOff};
+    const containerProps = {
+      'aria-describedby': this.tooltipId,
+      className: containerClassName,
+      ...hoverProps,
+    };
+    const hovercardProps = {
+      withHeader: !!header,
+      className: cx,
+      ...hoverProps,
+    };
```

Comment:
Can you get rid of these two objects and just inline the props in the components?